src/components/build/BuildSidebar.tsx

```diff
@@ -38,15 +43,8 @@ export class BuildSidebar extends React.Component<Props, State> {
     }
 
     const tasks = this.props.tasks.map(taskObj => (
-      <Grid item={true} xs={12} key={taskObj.task_id}>
-        <ExpansionPanel className="task-panel">
-          <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
-            <Typography>{taskObj.display_name}</Typography>
-          </ExpansionPanelSummary>
-          <ExpansionPanelDetails>
-            <Typography>No tests to show</Typography>
-          </ExpansionPanelDetails>
-        </ExpansionPanel>
+      <Grid item={true} xs={12} key={taskObj.display_name}>
```

Comment:
display_name isn't guaranteed to be unique. How come you switch from the task ID?

Reply:
The task ID is also not guaranteed to be unique. In fact, when I use my patch data, the task ID is the same for every element. Is there something else that would be better to use than the display name?

Reply:
Here's an example of what I get back as a response currently, where all task IDs are the same: https://evergreen.mongodb.com/api/rest/v2/tasks/mongodb_mongo_master_enterprise_rhel_62_64_bit_aggregation_auth_patch_aa9f6a202e0709adf14046cb27504864adaf732b_5d28d45fd1fe0716daa6b903_19_07_12_18_41_36/tests

Reply:
Oh wait... I see what you mean. I think this was a mistaken change, I thought the warning about unique IDs was pointing to this mapping of elements, but I think it was actually warning me about the tests in each task (it is very hard to determine the source for that warning). You are correct that I should be using the task IDs here and that they should be unique. Sorry for the misunderstanding.